igor-web/src/test/groovy/com/netflix/spinnaker/igor/build/BuildControllerSpec.groovy

```diff
@@ -106,7 +101,7 @@ class BuildControllerSpec extends Specification {
             .accept(MediaType.APPLICATION_JSON)).andReturn().response
 
         then:
-        3 * buildMasters.map >> [MASTER: service]
+        3 * buildMasters.map >> [(MASTER): service]
```

Comment:
I made a bunch of changes like this as it appears it was only working by accident before because the variable name `MASTER` is the same as its value. If you don't use braces here Groovy treats the key portion as a string literal _not_ a variable reference.